### ISSUE TITLE
How to change the iteration count for  breakpoint training

### ISSUE THREAD

**Issue by cainiao123s** [ultralytics/ultralytics#18640]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hello, I would like to ask some questions. I fixed the random seed to ensure that the experiment can be reproduced. During the training process, I initially set the epochs to 100, but after the training ended, the accuracy did not meet the requirements. I want to implement checkpoint training and set the epochs to 200. How can I achieve this functionality?

**Comment by UltralyticsAssistant** [comment]
👋 Hello @cainiao123s, thank you for your interest in Ultralytics 🚀! We recommend checking out the [Docs](https://docs.ultralytics.com) where you can explore examples for both [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage. Many common questions and workflows, such as checkpoint training, may already be covered there.

If this is a ❓ Question, please provide as much detail as possible about your training setup, including your dataset, training logs, and relevant hyperparameters. Make sure to also review our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/) to ensure optimal performance.

If this is related to a 🐛 Bug, please include a [minimum reproducible example (MRE)](https://docs.ultralytics.com/help/minimum_reproducible_example/) to help us understand and resolve the issue effectively.

To get started with checkpoint training or modifying the training process:
- Ensure you are using the latest version of Ultralytics. You can upgrade using:
```bash
pip install -U ultralytics
```
- You can resume training from a previous checkpoint by specifying the `resume` argument. Check the [DOCS](https://docs.ultralytics.com/) or share your current training configuration for personalized guidance.

Join the Ultralytics community to engage with other users and share your experiences:
- 📞 Real-time chat: [Discord](https://discord.com/invite/ultralytics)
- 📚 Forums: [Discourse](https://community.ultralytics.com)
- 🔗 Conversations: [Reddit](https://reddit.com/r/Ultralytics)

## Environments

YOLO training can run seamlessly in verified environments like:
- 🟢 **Google Colab**, **Paperspace**, or **Kaggle** Notebooks:
  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- 🖥️ Cloud platforms like [GCP](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/) or [AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- 🐳 Docker environments. Use the official Ultralytics Docker image:
  <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all Ultralytics CI tests are passing, ensuring core functionality operates as expected across macOS, Windows, and Linux.

This is an automated response to guide you in the right direction 🔄. An Ultralytics engineer will review your question and provide further assistance soon. Let us know if anything is unclear or if you’d like further clarification. 😊

**Comment by cainiao123s** [comment]
I tried, but it probably cannot achieve this function.

**Comment by Y-T-G** [comment]
You can't continue a finished training
